cloud/smartos/imgadm

```diff
@@ -0,0 +1,146 @@
+#!/usr/local/bin/python
+# -*- coding: utf-8 -*-
+
+# © 2014, Hugo Silva <freebsd.consulting@gmail.com>
+
+DOCUMENTATION = '''
+---
+module: imgadm
+author: Hugo Silva
+version_added: 1.8.2
+short_description: Manage SmartOS VM images
+description:
+    -  The imgadm tool allows you to interact with virtual machine images on a
+       SmartOS system. Virtual machine images (also sometimes referred to as
+       'datasets') are snapshots of pre-installed virtual machines which are
+       prepared for generic and repeated deployments.
+options:
+    command:
+        required: true
+        choices: [sources, import, update]
```

Comment:
You mention "import" in help, but use "image" in the code and examples. This should be synced.